test/check-application

```diff
@@ -2122,6 +2122,19 @@ class TestApplication(testlib.MachineCase):
         self.execute(auth, f"podman inspect --format '{{{{.Id}}}}' {container_name_new}").strip()
         self.waitContainerRow(container_name_new)
 
+    def testMultipleContainers(self):
+        self.login()
+
+        # Create 31 containers
+        for i in range(31):
+            self.execute(True, f"podman run -dt --name container{i} quay.io/libpod/busybox:latest")
+
+        self.waitContainerRow("container30")
```

Comment:
So the race condition would have broken the protocol persistently? I.e. why does it only check for container30, but not for the 29 other ones?

Reply:
It will happen if the page is opened with more than ~15 containers as it get update for stats. I just put there wait for the last container to appear, that gives the page enough open time for the oops to appear. And the oops is what kills the test.

Reply:
I see, thanks!